flocker/testtools/_flaky.py

```diff
@@ -4,8 +4,19 @@
 Logic for handling flaky tests.
 """
 
+from functools import partial
+from pprint import pformat
 
-def flaky(jira_key):
+from pyrsistent import PClass, field, pmap, pset, pset_field
+import testtools
+from testtools.content import text_content
+from testtools.testcase import gather_details
+
+
+_FLAKY_ATTRIBUTE = '_flaky'
+
+
+def flaky(jira_keys, max_runs=5, min_passes=2):
```

Comment:
Actually, we haven't had those conversations. I picked those numbers based on the upstream box/flaky defaults.

Changed to your suggested defaults.